packages/auth/src/api/password_policy/get_password_policy.ts

```diff
@@ -43,6 +43,8 @@ export interface GetPasswordPolicyResponse {
     containsNonAlphanumericCharacter?: boolean;
   };
   allowedNonAlphanumericCharacters: string[];
+  enforcementState: string;
+  forceUpgradeOnSignin?: boolean;
```

Comment:
why optional?

Reply:
This field is optional in PasswordPolicyConfig on the backend, meaning it could be undefined. I could change this to be non-optional but would still need the undefined check inside PasswordPolicyImpl.